tests/test_helpers.py

```diff
@@ -33,10 +34,6 @@ def test_parse_areas_config_raises_value_error():
         parse_areas_config("3.4", raises=True)
 
 
-def test_parse_areas_config_whitespace():
```

Comment:
I think you should remove also other tests as they are testing a format that is invalid. Basically, remove tests that are testing numbers. Keep the ones that are testing an empty list, `None` and so on.

Reply:
We don't need these tests anymore! 🎉